Source/Service/Requests/CreateAuthenticationEvent.swift

```diff
@@ -0,0 +1,105 @@
+//
+//  CreateAuthenticationEvent.swift
+//  Cotter
+//
+//  Created by Albert Purnama on 3/1/20.
+//
+
+import Foundation
+
+public struct CreateAuthenticationEvent: APIRequest {
+    public typealias Response = CotterEvent
+    
+    public var path: String {
+        return  "/event/create"
+    }
+
+    public var method: String {
+        return "POST"
+    }
```

Comment:
same

Reply:
I will change this to let since it is constant